release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -167,6 +167,33 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-networking-egress-service"]
+==== Egress service resource to manage egress traffic for pods behind a load balancer (Technology Preview)
+
+With this update, you can use an `EgressService` custom resource (CR) to manage egress traffic for pods behind a load balancer service. This is available as a Technology Preview feature.
+
+You can use the `EgressService` CR to manage egress traffic in the following ways:
+
+* Assign a load balancer service's IP address as the source IP address for egress traffic for pods behind the load balancer service.
```

Comment:
`Assign a` -> `Assign the`?
`for egress traffic for` -> `of egress traffic for` / `for egress traffic of` ?